src/mid.cpp

```diff
@@ -943,19 +966,38 @@ void CmidPlayer::rewind(int subsong)
             case FILE_MIDI:
                 if (type != FILE_LUCAS)
                     tins=128;
-                getnext(11);  /*skip header*/
+                getnext(11); /* skip past header data until deltas is reached */
                 deltas=getnext(2);
                 midiprintf ("deltas:%ld\n",deltas);
-                getnext(4);
+	        for (i=0; i<16; i++)
+	        {
+                  ch[i].nshift=-13;
+                  ch[i].on=1;
```

Comment:
Also this change is not clear for me, why we need to mark all channels as "on"?

I have tested with several MIDI files, it doesn't have any audible effect to me.